src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -405,10 +404,6 @@ public void hoverClickAndViewPhotoOnTableCell(int questionBodyIndex, int tableRo
         
         waitForElementPresence(By.cssSelector(".popover-content"));
         
-        WebElement popoverContent = browser.driver.findElement(By.className("popover-content"));
-        
-        waitForElementVisibility(popoverContent);
-        
```

Comment:
It's already working. All one needs is waiting for presence, which has already been done. I assume from the logs that #4797 and alike happens because of the short timeout used (in live server 15 secs won't be enough).